src/sentry/models/projectownership.py

```diff
@@ -84,7 +84,15 @@ def get_owners(cls, project_id, data):
             return cls.Everyone if ownership.fallthrough else [], None
 
         owners = {o for rule in rules for o in rule.owners}
-        return filter(None, resolve_actors(owners, project_id).values()), rules
+        owners_to_actors = resolve_actors(owners, project_id)
+        ordered_actors = []
+        for rule in rules:
+            for o in rule.owners:
+                if o in owners and owners_to_actors[o] is not None:
```

Comment:
Could the `o` key be undefined at this point? Or will it always exist?

Reply:
I don't think so but it's easy to protect against.